src/ServicePulse.Host/vue/src/components/monitoring/LargeGraph.vue

```diff
@@ -0,0 +1,174 @@
+<script setup>
```

Comment:
it is worthwhile doing a diff between this file and D3LargeGraph rather than looking at it as a brand new file